msteams-platform/get-started/glossary.md

```diff
@@ -45,7 +45,7 @@ Common terms and definitions used in Teams developer documentation.
 | [Capability](../toolkit/add-capability.md) | A Teams feature you can build into your app for interacting with app users. An app capability is used to extend Teams to fit your app needs. An app may have one or more core capabilities, such as tab, bot, and message extension. <br>**See also**: [Device capability](#d); [Media capability](#m) |
 | [Chat bot](../bots/how-to/conversations/conversation-basics.md) | A bot is also referred to as a chatbot or conversational bot. It's an app that runs simple and repetitive tasks for users such as customer service or support staff. <br> **See also**: [Conversational bot](#c) |
 | Channel | A single place for a team to share messages, tools, and files. You can use a channel for teamwork and communication. <br>**See also**: [Conversation](#c) |
-| [Client secret](../bots/how-to/authentication/add-authentication.md) | The Client secret/password or a public or private key pair that is Certificate. It isn't required for native apps. <br> **See also**: [Bot](#b) |
+| [Client secret](../bots/how-to/authentication/add-authentication.md) | A secret string that the application uses to prove its identity when requesting a token. Also can be referred to as application password.|
```

Comment:
`Also, it can be referred to as application password.`
OR 
`It is also referred to as application password.`
OR 
`Also called as application password.`